packages/mobile/lib/routers/routers/settings-router.ts

```diff
@@ -0,0 +1,56 @@
+import { get, writable, Writable } from 'svelte/store'
```

Comment:
There is a settings-router left in the dashboard folder, which I've created with my last PR. I guess we should remove it.